docs/src/pages/demos/lists/NestedList.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import ListSubheader from 'material-ui/List/ListSubheader';
+import List, { ListItem, ListItemIcon, ListItemText } from 'material-ui/List';
+import Collapse from 'material-ui/transitions/Collapse';
+import InboxIcon from 'material-ui-icons/MoveToInbox';
+import DraftsIcon from 'material-ui-icons/Drafts';
+import SendIcon from 'material-ui-icons/Send';
+import KeyboardArrowUp from 'material-ui-icons/KeyboardArrowUp';
+import KeyboardArrowDown from 'material-ui-icons/KeyboardArrowDown';
```

Comment:
Should use `ExpandLess` and `ExpandMore`.  Same icon, but the name is more appropriate:

```
import ExpandLessIcon from 'material-ui-icons/ExpandLess';
import ExpandMoreIcon from 'material-ui-icons/ExpandMore';
```